apps/dashboard/src/data/ui/actions.js

```diff
@@ -53,6 +54,7 @@ export const infoNotice = ( message, { id, ...options } ) =>
 	createNotice( id || uniqueId(), 'info', message, options );
 
 export function redirect( path ) {
```

Comment:
We won't be needing this action anymore either, let's just remove it along with its action type.  
Just as you did here, you can call `routeRedirect()` directly _anywhere_ hence no need to do the whole wp.data dance for it.